services/portal/studyViewer/studyViewerTasks.js

```diff
@@ -38,10 +39,29 @@ module.exports = {
     await I.seeElement(studyViewerProps.studyViewerDivClass);
   },
 
+  // testing multiple datasets
+  async multipleStudyViewer() {
```

Comment:
"test multiple study viewers" as mentioned in the test plan means having multiple study viewer pages configured (reading from different ES indices). I see here https://github.com/uc-cdis/gitops-qa/pull/1237 that only one study viewer is configured, so I think the "test multiple study viewers" scenario will be in a future iteration. If that's the case, could you please rename this function to match the scenario name ("Multiple dataset")

Reply:
Yes! good catch! It was skipped by me in final check before pushing the commit